yubikey-luks-enroll

```diff
@@ -51,10 +51,20 @@ if [ $CLEAR_SLOT -eq 1 ]; then
 fi
 
 echo "Adding yubikey to initrd"
-P1=$(/lib/cryptsetup/askpass "Please insert a yubikey and enter a new password. This is the password that will only work while your yubikey is installed in your computer.")
+
+while true ; do
+    if lsusb | grep -iq 'yubico'; then break; fi
+    printf "Please insert a yubikey and press enter."
+    read -r _ <&1
+done
+
+I=
+if [ "$Y" != "0" ]; then I=' and press the button on the Yubikey when flashing'; fi
```

Comment:
At this point you could somehow adapt the WELCOME_TEXT.

Reply:
Seems like this is a leftover from a previous iteration of the code (probably when using `ykchalresp -N` to test for block on touch). I'll remove this as there isn't really a way of testing (`ykchalresp -N` breaks subsequent `ykchalresp` calls for the timeout period - see PR description).